src/systems/triggered_publisher/TriggeredPublisher.cc

```diff
@@ -598,31 +612,73 @@ void TriggeredPublisher::Configure(const Entity &,
   else
   {
     ignerr << "No ouptut specified" << std::endl;
-    return;
+  }
+
+  if (sdfClone->HasElement("service"))
+  {
+    for (auto serviceElem = sdfClone->GetElement("service"); serviceElem;
+         serviceElem = serviceElem->GetNextElement("service"))
+    {
+      ServiceOutputInfo serviceInfo;
+      serviceInfo.servName = serviceElem->Get<std::string>("name");
+      if (serviceInfo.servName.empty())
+      {
+        ignerr << "Service name cannot be empty\n";
+        return;
```

Comment:
This means we will skip any further setup here is any of the service names are invalid.  Going with what the outputs above do, I think this might be better as a `continue`.